src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/coordinator/SweptManagedIndexConfig.kt

```diff
@@ -35,7 +40,22 @@ data class SweptManagedIndexConfig(
     val uuid: String,
     val policyID: String,
     val changePolicy: ChangePolicy?
-) {
+): ToXContentObject {
+
+    /**
+     * Used in the partial update of ManagedIndices when calling the
+     * [RestChangePolicyAction] API
+     */
+    override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
```

Comment:
This doesn't feel right on how we implement `XContent`. This implementation only gives partial information about this data class where usually `toXContent` gives all the information about class so it can be re-parsed as the data class. With this `toXContent` we will not be able to recreate the object. 

Can we create separate function to only get subset of information to `XContent`?

Reply:
Will move into a helper function